.pipelines/templates/enable-arc.yaml

```diff
@@ -1,10 +1,16 @@
 steps:  
+  - bash: |
+      az extension remove --name connectedk8s
```

Comment:
can we keep these commands in the shell script instead of here?

Reply:
I needed to move these cmds here in order to keep the connect-to-arc.sh runnable locally. 

AZ_EXTENSION_LOCATION is a secret variable, and we can't use it in the connect-to-arc script unless we pass it in. The problem with passing it in is that we can't assign it a default value, because the storage account is supposed to be hidden. So this will render that script unusable locally.

Reply:
Users will have to have connectedk8s installed beforehand in order to run the connect-to-arc script now.

Reply:
Cloning into a private repo inside a script to add an extension also isn't an option because the users will need to clone that repo inside of osm-azure.

Reply:
I think it's reasonable to have the user running the connect-to-arc script to set `AZ_EXTENSION_LOCATION` like all other environment variables. These scripts are meant for devs and we can assume devs should have access to the whl files.